src/main/java/core/basesyntax/LambdaCalculator.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class LambdaCalculator {
+    public double calculate(int firstValue, int secondValue, char operation) {
+        if (operation == '+') {
+            return firstValue + secondValue;
+        }
+        if (operation == '-') {
+            return firstValue - secondValue;
+        }
+        if (operation == '*') {
+            return firstValue * secondValue;
+        }
+        if (operation == '/') {
+            if (secondValue == 0) {
+                throw new ArithmeticException("We cant divide by 0");
+            }
+            return (double) firstValue / secondValue;
+        }
+        if (operation == '^') {
+            return Math.pow(firstValue, secondValue);
```

Comment:
I think we should implement own implementation, but I don't know))
```suggestion
            return Math.pow(firstValue, secondValue);
```

Reply:
@Fedich09 we can use Math.pow() here)